diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6330d56..6336b6b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,9 +58,6 @@ find_package(
         SyntaxHighlighting
 )
 
-find_package(KUserFeedback)
-set_package_properties(KUserFeedback PROPERTIES TYPE REQUIRED PURPOSE "Checking whether feedback is enabled or not")
-
 ecm_find_qmlmodule(org.kde.kirigami 2.19)
 ecm_find_qmlmodule(org.kde.kitemmodels 1.0)
 ecm_find_qmlmodule(org.kde.kcm 1.6)
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index fb03182..5f1a091 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -103,7 +103,6 @@ target_link_libraries(
     drkonqi_backtrace_parser
     qbugzilla
     KF5::Declarative
-    KUserFeedbackCore
 )
 
 if(Systemd_FOUND)
diff --git a/src/bugzillaintegration/reportinterface.cpp b/src/bugzillaintegration/reportinterface.cpp
index 6951a75..bc9770c 100644
--- a/src/bugzillaintegration/reportinterface.cpp
+++ b/src/bugzillaintegration/reportinterface.cpp
@@ -14,7 +14,6 @@
 
 #include <KIO/TransferJob>
 #include <KLocalizedString>
-#include <KUserFeedback/Provider>
 
 #include "backtracegenerator.h"
 #include "bugzillalib.h"
@@ -58,13 +57,6 @@ ReportInterface::ReportInterface(QObject *parent)
         m_sentryUserFeedbackSent = true;
         maybeDone();
     });
-    if (KUserFeedback::Provider provider; provider.isEnabled() && provider.telemetryMode() == KUserFeedback::Provider::DetailedUsageStatistics
-        && !DrKonqi::isTestingBugzilla() && qgetenv("DRKONQI_KDE_BUGZILLA_URL").isEmpty() && !DrKonqi::crashedApplication()->hasDeletedFiles()) {
-        metaObject()->invokeMethod(this, [this] {
-            // Send crash event ASAP, if applicable. Trace quality doesn't matter for it.
-            sendCrashEvent();
-        });
-    }
 }
 
 void ReportInterface::setBugAwarenessPageData(bool rememberSituation, Reproducible reproducible, bool actions, bool unusual, bool configuration)
